src/screenshot.cpp

```diff
@@ -121,6 +121,39 @@ void Screenshot::savePathScreenshot(const QString &path)
     m_window->savePath(path);
 }
 
+void Screenshot::multioptionalScreenshot(int areaOption, QString savePath){
+    initUI();
```

Comment:
you don't need 

```
initUI();
this->show();
```

it's already int `fullScreenshot()` and `topWindow()`;

Reply:
I tried to comment the two lines, but it just ran incorrectly. Perhaps there are different functions between ```Screenshot::show()``` and ```MainWindow::show()```